<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>866</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>同步V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>39</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbDatebase">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>37</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Test">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>199</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>71</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库地址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>105</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库名称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>137</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库用户：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>169</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ET_ip">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>70</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ET_DbName">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>104</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ET_User">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>136</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ET_PassWord">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>167</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Save">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>199</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>记住设置</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tab_BD">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>6</y>
       <width>371</width>
       <height>230</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>主数据库：              同步到：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_AddDB">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>243</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>增加数据库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_DelDB">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>243</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除数据库</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>1090</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>规则设置(蓝色可为空)---------------------------------------------------------------------------------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>同步到：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbDbInfoList">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>317</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>317</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入表名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_table">
    <property name="geometry">
     <rect>
      <x>623</x>
      <y>317</y>
      <width>180</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>388</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更新列：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbbTransfield">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ET_SQL">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>455</y>
      <width>100</width>
      <height>90</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_UpdateName">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>387</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddSQL">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>418</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>添加更新</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ET_operatorText">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>430</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>422</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更新值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_UpdateValue">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>422</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>546</y>
      <width>1100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更新规则明细------------------------------------------------------------------------------------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddRule">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>513</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>加入规则</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>353</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>唯一键：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_Key">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>353</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_Where">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>479</y>
      <width>200</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>476</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ET_showUpdate">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>454</y>
      <width>199</width>
      <height>90</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>318</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更新间隔：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_SleepTime">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>318</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableRule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>598</y>
      <width>1090</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_QryRule">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>563</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Delete">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>563</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_GetColName">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>312</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>获取字段</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ET_ColName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>347</y>
      <width>570</width>
      <height>80</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>916</x>
      <y>318</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>同步字段的顺序需一致</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_GetColName_Tag">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>432</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>获取字段</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_table_Tag">
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>437</y>
      <width>180</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ET_ColName_Tag">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>467</y>
      <width>570</width>
      <height>80</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>437</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>同步到表名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>422</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>（可多次）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ChangeCols">
    <property name="geometry">
     <rect>
      <x>919</x>
      <y>432</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>同步到不同字段时</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TransStart">
    <property name="geometry">
     <rect>
      <x>926</x>
      <y>563</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开始同步</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>353</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>自增列：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_autoCol">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>353</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>353</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>序列：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>321</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>序列仅Oracle生效</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_seq">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>353</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>390</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>特殊列：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_SpecialCol">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>389</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>323</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>特殊列规则：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ET_SpecialRule">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>451</y>
      <width>190</width>
      <height>90</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Check">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>414</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>验证</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_TransStop">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>562</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>737</x>
      <y>568</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更新条数/次：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ET_nRecrod">
    <property name="geometry">
     <rect>
      <x>837</x>
      <y>568</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
